ui/src/app/components/webcomponents/test-step-form-container.component.ts

```diff
@@ -309,8 +309,13 @@ export class TestStepFormContainerComponent extends BaseComponent implements OnI
     }
     testStep.stepDisplayNumber = this.indexPosition(false);
     if(testStep.preRequisiteStepId) {
-      this.testSteps.content.find(testStep => testStep.id == this.testStep.preRequisiteStepId)
-        .stepDisplayNumber = this.indexPosition(true);
+      let preRequisiteStep = this.testSteps.content.find(testStep => testStep.id == this.testStep.preRequisiteStepId)
+        if(preRequisiteStep){
+          preRequisiteStep.stepDisplayNumber = this.indexPosition(true);
```

Comment:
what is the return of this.indexPosition(true); 
is it a number?

Reply:
yes it returns number